app/models/manageiq/providers/redhat/infra_manager/inventory/strategies/v4.rb

```diff
@@ -98,7 +98,7 @@ def collect_templates
     end
 
     def search_templates(search)
-      connection.system_service.templates_service.list(:search => search).collect do |template|
+      conneion.system_service.templates_service.list(:search => search).collect do |template|
```

Comment:
We need to fix it. I looked for it before but thought it was fixed. Now I found it.